### ISSUE TITLE
Better to have a controller of its own for the camera

### ISSUE THREAD

**Issue by axaq** [pmndrs/racing-game#14]
Hi, firstly I loved this proof of consepting idea. 

I have worked on a lot of web games and in almost every case, we ended up creating a controller for the camera since down the line you want specific camera movements for certain cases. Putting the logic in vehicle and or track easily becomes messy and harder to follow. 

So my proposition is to create a separate controller for camera and handle its movements there in accordance with vehicle and track.

I can make a start and create a PR soon, if you like the idea.

**Comment by drcmda** [comment]
yes, please do. i have no idea what i'm doing and controlling the camera is the hardest part for me due to the math.

**Comment by ManVTish** [comment]
Can anyone please help me with getting started with solving issues. I'm new to open source community and projects. I do have a background in coding though. Please help me getting started
